packages/docs/src/pages/en/components/editor.md

```diff
@@ -0,0 +1,122 @@
+---
+emphasized: true
+meta:
+  nav: WYSIWYG Editor
+  title: Editor component
+  description: The editor component is a rich text editor that allows users to create and edit content with formatting options.
+  keywords: editor, vuetify editor component, vue editor component, rich text editor, wysiwyg
+related:
+  - /components/textareas/
+  - /components/text-fields/
+  - /components/forms/
+features:
+  github: /labs/VEditor/
+  label: 'C: VEditor'
+  report: true
+---
+
+# WYSIWYG Editor
+
+The `v-editor` component provides a rich text editor with formatting capabilities, allowing users to create and edit content with text formatting options.
+
+<PageFeatures />
+
+::: warning
+
+This feature requires [v3.9.6](/getting-started/release-notes/?version=v3.9.6)
+
+:::
+
+## Installation
+
+Labs components require manual import and registration with the Vuetify instance.
+
+```js { resource="src/plugins/vuetify.js" }
+import { VEditor } from 'vuetify/labs/VEditor'
+
+export default createVuetify({
+  components: {
+    VEditor,
+  },
+})
+```
+
+## Usage
+
+The `v-editor` component combines the functionality of a text area with a toolbar for text formatting. It supports basic rich text editing with a clean, intuitive interface.
+
+<ExamplesUsage name="v-editor" />
+
+<PromotedEntry />
+
+## API
+
+| Component | Description |
+| - | - |
+| [v-editor](/api/v-editor/) | Primary Component |
+
+<ApiInline hide-links />
+
+## Guide
+
+The `v-editor` component is built on top of Vuetify's `v-input` and `v-field` components, providing a rich text editing experience. It includes a toolbar with formatting buttons and a content-editable area for text input.
+
+### Props
+
+The `v-editor` component inherits all props from `v-input` and `v-field` components, plus its own specific props for editor functionality.
+
+#### Basic usage
+
+A simple editor with default toolbar (bold, italic, underline):
+
+<ExamplesExample file="v-editor/prop-basic" />
+
+#### Custom toolbar items
+
+You can customize which formatting options are available in the toolbar using the `formats` prop:
+
+<ExamplesExample file="v-editor/prop-formats" />
+
+#### Hide toolbar
+
+You can hide the formatting toolbar entirely using the `hide-toolbar` prop:
```

Comment:
It would be worth mentioning that formatting can then be achieved with keyboard shortcuts or by pasting the formatted text.

Example text (in the field) could be adjusted. "..._without formatting options._" is a bit confusing.